pangeo_forge_recipes/writers.py

```diff
@@ -95,23 +97,47 @@ def store_dataset_fragment(
 def write_combined_reference(
     reference: MultiZarrToZarr,
     full_target: FSSpecTarget,
-    output_json_fname: str,
-):
+    concat_dims: List[str],
+    output_file_name: str,
+    refs_per_component: int = 1000,
+) -> FSSpecTarget:
     """Write a kerchunk combined references object to file."""
 
     import ujson  # type: ignore
 
-    multi_kerchunk = reference.translate()
-    file_ext = os.path.splitext(output_json_fname)[-1]
+    file_ext = os.path.splitext(output_file_name)[-1]
+
+    outpath = full_target._full_path(output_file_name)
 
     if file_ext == ".json":
-        outpath = os.path.join(full_target.root_path, output_json_fname)
+        multi_kerchunk = reference.translate()
         with full_target.fs.open(outpath, "wb") as f:
             f.write(ujson.dumps(multi_kerchunk).encode())
+
+    elif file_ext == ".parquet":
+
+        # Creates empty parquet store to be written to
+        if full_target.exists(output_file_name):
+            full_target.rm(output_file_name, recursive=True)
+        full_target.makedir(output_file_name)
+
+        # kwargs to pass to MultiZarrToZarr
+        fs = fsspec.filesystem("file")
+        out = LazyReferenceMapper.create(refs_per_component, outpath, fs)
```

Comment:
```suggestion
        out = LazyReferenceMapper.create(refs_per_component, outpath, full_target.fs)
```

I think we need the `full_target.fs` instead of `fsspec.filesystem("file")` here, since we want to be able to write to remote storage?